src/commands/sites/sites-create-template.js

```diff
@@ -80,13 +80,20 @@ const sitesCreateTemplate = async (repository, options, command) => {
       `Could not find template ${chalk.bold(templateName)}. Please verify it exists and you can ${terminalLink(
         'access to it on GitHub',
         githubLink,
+        {
+          fallback: () => `access to it on GitHub ${githubLink}`,
+        },
       )}`,
     )
     return
   }
   if (!isTemplate) {
     const githubLink = getGitHubLink({ options, templateName })
-    error(`${terminalLink(chalk.bold(templateName), githubLink)} is not a valid GitHub template`)
+    error(
+      `${terminalLink(chalk.bold(templateName), githubLink, {
+        fallback: () => `${chalk.bold(templateName)} ${githubLink}`,
+      })} is not a valid GitHub template`,
```

Comment:
What are your thoughts on using a shared utility function to reduce code duplication? For example:

```js
const getTerminalLink = (text, url) => terminalLink(text, url, { fallback: () => `${text} ${url}` })
```

Reply:
It's definitely a good idea, I'll implement it. I'll move it to a util file as well, in case any future changes to it are needed, we'll only need to do it in one place..